includes/Modules/Analytics.php

```diff
@@ -518,7 +519,24 @@ protected function create_data_request( Data_Request $data ) {
 					return true;
 				};
 			case 'GET:accounts-properties-profiles':
-				return $this->get_service( 'analytics' )->management_accounts->listManagementAccounts();
+				$restore = $this->with_client_defer( false );
```

Comment:
Let's move this into the anonymous function below. It should only change the client defer mode right before the request, here that _may_ not be the case depending on how the surrounding code deals with the returned function.

Also, I think it would be better naming the variable something more precise (we're using `$restore_defer` in other cases).